src/Zoom.js

```diff
@@ -26,11 +26,11 @@ export default class Zoom extends Component {
   }
 
   componentDidMount() {
-    const { hasAlreadyLoaded, zoomImage: { src, srcSet } } = this.props
+    const { zoomImage: { src, srcSet } } = this.props
 
     this.setState({ hasLoaded: true })
 
-    if ((src || srcSet) && !hasAlreadyLoaded) {
+    if (src || srcSet) {
```

Comment:
The original thinking with this was that if the image had already been downloaded, then don't bother doing the whole `new Image()` thing again; however, that is _very_ cheap as it fetches from the browser's cached version. 

This is where the bug came from.

If the `Zoom` component had already been zoomed and unzoomed, then the parent component, `ImageZoom` set this arbitrary `hasAlreadyLoaded` property. However, in the case where we _do not replace the original image_ and we _have a zoomImage_, it would prevent the zoomImage from being displayed after the first time.